### ISSUE TITLE
coverage combine doesn't work with rcfile

### ISSUE THREAD

**Issue by nedbat** [coveragepy/coveragepy#385]
*Originally reported by* **Anonymous**

----------------------------------------

Hi,

I have plenty of .coverage.* files and trying to combine them by running:
% coverage combine --rcfile .coveragerc
Where .coveragerc looks like:
[paths]
my_dirs =
        <path 1>
        <path 2>

It used to work with 4.0a5 but not working with 4.0a6

Thanks,
Maoz

----------------------------------------
- Bitbucket: https://bitbucket.org/ned/coveragepy/issue/385
- This issue had attachments: coverage_bug.tar.gz.  See the [original issue](https://bitbucket.org/ned/coveragepy/issue/385) for details.

**Comment by nedbat** [comment]
Can you provide more details about how it isn't working? Can you give me the exact contents of your rcfile?

**Comment by nedbat** [comment]
*Original comment by* **Maoz Guttman (Bitbucket: [maozguttman](https://bitbucket.org/maozguttman), GitHub: [maozguttman](https://github.com/maozguttman))**

----------------------------------------

Hi,

By saying not working, I mean that none of the .coverage.* files were combined into .coverage file. They were also not deleted after running "coverage combine".

.coveragerc file content is as following, where <path n> is a full path pointer to a directory in NFS

```
#!Text

[paths]
my_dirs =
        <path 1>
        <path 2>
        <path 3>
        <path 4>


```
p.s. I am running it on Linux (Suse10).

Thanks,
Maoz

**Comment by nedbat** [comment]
@maozguttman Thanks.  Do you know if NFS is a key factor here?  Have you tried a similar operation not on NFS?  Do you mind pasting the exact literal text of the .coveragerc file? I'm looking for any clue at all... :)

**Comment by nedbat** [comment]
*Original comment by* **Maoz Guttman (Bitbucket: [maozguttman](https://bitbucket.org/maozguttman), GitHub: [maozguttman](https://github.com/maozguttman))**

----------------------------------------

It is not related to NFS. I was trying it on /tmp local disk.
.coverage file is generated when running "coverage combine", but it seems to be empty since "coverage html" reports "No data to report." Also its size is 64B comparing to 10KB of the raw .coverage.* files.

Here is rc file:
```
#!text
[paths]
my_dirs =
        /tmp/maozg/casa_main_branch_1/out/em64t_SLES10/st64

```

**Comment by nedbat** [comment]
*Original comment by* **Maoz Guttman (Bitbucket: [maozguttman](https://bitbucket.org/maozguttman), GitHub: [maozguttman](https://github.com/maozguttman))**

----------------------------------------

Do you want me to create a toy example for you? or can you reproduce it without it?

Thanks,
Maoz

**Comment by nedbat** [comment]
@maozguttman if you could make an example, that would be amazing, thanks!

**Comment by nedbat** [comment]
*Original comment by* **Maoz Guttman (Bitbucket: [maozguttman](https://bitbucket.org/maozguttman), GitHub: [maozguttman](https://github.com/maozguttman))**

----------------------------------------

How to reproduce:

```
#!tcsh
gtar -xzvf coverage_bug.tar.gz
cd coverage_bug
./run.tcsh

```


Bug: run above test case with both 4.05a and 4.06a versions. "coverage combine" works in 4.05a, but not in 4.06a

**Comment by nedbat** [comment]
*Original comment by* **Maoz Guttman (Bitbucket: [maozguttman](https://bitbucket.org/maozguttman), GitHub: [maozguttman](https://github.com/maozguttman))**

----------------------------------------

Hi,

I attached a test case.

How to reproduce:

```
#!tcsh
gtar -xzvf coverage_bug.tar.gz
cd coverage_bug
./run.tcsh

```

Bug: run above test case with both 4.05a and 4.06a versions. "coverage combine" works in 4.05a, but not in 4.06a


Thanks,
Maoz

**Comment by nedbat** [comment]
@maozguttman Thanks! That made it very clear.  Fixed in [0007fc5762b6 (bb)](https://bitbucket.org/ned/coveragepy/commits/0007fc5762b6)

**Comment by nedbat** [comment]
*Original comment by* **Maoz Guttman (Bitbucket: [maozguttman](https://bitbucket.org/maozguttman), GitHub: [maozguttman](https://github.com/maozguttman))**

----------------------------------------

Tnx!

Maoz
